core/validatorapi/teku.go

```diff
@@ -36,7 +36,10 @@ type TekuBuilder struct {
 	Overrides map[string]string `json:"registration_overrides,omitempty"`
 }
 
-const gasLimit = 30000000
+const (
+	gasLimit    = 30000000
+	zeroAddress = "0x0000000000000000000000000000000000000000"
```

Comment:
zero values mean golang zero values, so do not populate the `Default`field. Just make sure Teku validation doesn't fail in this case.

Reply:
yup, tried that earlier unfortunately teku doesn't like that:
![Screenshot 2023-01-25 at 4 23 41 PM](https://user-images.githubusercontent.com/37813203/214545409-70ab7147-62f8-4950-926b-c0f9939c526f.png)